shop/src/main/java/ro/msg/learning/shop/domain/Keys/StockKey.java

```diff
@@ -0,0 +1,18 @@
+package ro.msg.learning.shop.domain.Keys;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embeddable;
+import lombok.Data;
+
+import java.io.Serializable;
+import java.util.UUID;
+
+@Data
+@Embeddable
+public class StockKey implements Serializable {
+    @Column(name = "product")
+    private UUID product;
+
+    @Column(name = "location")
+    private UUID location;
```

Comment:
These fields should be better renamed to `productId` and `locationId`. 

Do the same for the other `UUID` fields in the application.